manifests/base/cluster-role.yaml

```diff
@@ -48,24 +48,24 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: kubeflow-tfjobs-admin
+  name: kubeflow-training-admin
   labels:
     rbac.authorization.kubeflow.org/aggregate-to-kubeflow-admin: "true"
 aggregationRule:
   clusterRoleSelectors:
   - matchLabels:
-      rbac.authorization.kubeflow.org/aggregate-to-kubeflow-tfjobs-admin: "true"
+      rbac.authorization.kubeflow.org/aggregate-to-kubeflow-training-admin: "true"
 rules: []
 
 ---
 
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: kubeflow-tfjobs-edit
+  name: kubeflow-training-edit
   labels:
     rbac.authorization.kubeflow.org/aggregate-to-kubeflow-edit: "true"
-    rbac.authorization.kubeflow.org/aggregate-to-kubeflow-tfjobs-admin: "true"
+    rbac.authorization.kubeflow.org/aggregate-to-kubeflow-training-admin: "true"
 rules:
 - apiGroups:
```

Comment:
I think you should add other roles here also.
As @Jeffwan mentioned: https://github.com/kubeflow/tf-operator/pull/1346#discussion_r688093994.

Reply:
Let's keep `/config` folder now. We may still modify codes and generate new manifests. I think we need to consider how to maintain source of truth in the future. We can also write some guidance to show how to maintain manifests. For example, we update apis and generate new crds, they should be generated and move to /manifests/crd/ etc.

Reply:
@Jeffwan I guess we can point `kubebuilder` where we want to have our manifests here: https://github.com/kubeflow/tf-operator/blob/master/Makefile#L41, right ?
For example: 
```
$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./pkg/apis/..." output:crd:artifacts:config=manifests/base
```

Reply:
I agree, eventually we should move to using above auto generation. also @andreyvelich i have added resources in respective cluster roles but am unclear what needs to be added in labels for aggregates. respective roles already have training view, edit and admin labels where needed.

Reply:
yeah, that works for CRDs.  I can not remember if roles can be located to other folder as well.  If so, we can point to manifests directly. For rest of the files, we can just move there because kubebuilder won't track it anymore after first time generation.

Reply:
@deepak-muley

Can you address Andrey's comment? Let's remove config folder and point points to manifest/base. crd names needs to be changed to kubebuilder fashion. 

Another change to make is  setup-training-operator.sh. CI can use `/manifest` instead.

Reply:
>i have added resources in respective cluster roles but am unclear what needs to be added in labels for aggregates. respective roles already have training view, edit and admin labels where needed.

@deepak-muley I think we should remove these roles from [the default ClusterRole](https://github.com/kubeflow/tf-operator/blob/833c792fa8b74b2f90493d4342ddb5f9aef72bd4/manifests/base/cluster-role.yaml#L48-L102).
This relates only for the Kubeflow installation.
Please can you update Kubeflow kustomize file with the similar resource: https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/installs/katib-with-kubeflow/kustomization.yaml#L7.
And create separate file under `manifests/overlays/kubeflow` with the Kubeflow roles: https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/installs/katib-with-kubeflow/kubeflow-katib-roles.yaml.

Reply:
> @deepak-muley
> 
> Can you address Andrey's comment? Let's remove config folder and point points to manifest/base. crd names needs to be changed to kubebuilder fashion.
> 
> Another change to make is setup-training-operator.sh. CI can use `/manifest` instead.

will be using /manifests/overlays/standalone here instead of config/default